File: src/main/java/com/example/desafio/api/models/dtos/BrandDto.java
package com.example.desafio.api.models.dtos;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class BrandDto {
    private Long id;
    private Boolean deleted;
    @NotBlank(message = "O nome da marca é obrigatório")
    private String nameBrand;
}

File: src/main/kotlin/com/example/desafio/api/models/dtos/BrandDto.kt
package com.example.desafio.api.models.dtos

import jakarta.validation.constraints.NotBlank

data class BrandDto(
        var id: Long? = null,

        var deleted: Boolean? = null,

        @field:NotBlank(message = "O nome da marca é obrigatório")
        var nameBrand: String? = null
)

